projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -60,8 +60,9 @@ public static function register_feature( $features ) {
 		// Features under development.
 		$features['image-select-field'] = apply_filters( 'forms_alpha', false );
 
-		// Features that are only available to users with a paid plan.
-		$features['multistep-form'] = Current_Plan::supports( 'multistep-form' );
+		// Features that are only available to users with a paid plan or on Big Sky trial sites.
+		$is_big_sky_free_trial      = function_exists( 'has_blog_sticker' ) && has_blog_sticker( 'big-sky-free-trial' );
```

Comment:
Looks like there's also `wpcomsh_is_site_sticker_active()` which might be relevant for Atomic sites.